content/en/blog/2025/contribex-survey-results/index.md

```diff
@@ -0,0 +1,278 @@
+---
+title: Insights from the OpenTelemetry Contributor Experience Survey
+linkTitle: ContribEx survey results
+date: 2025-06-09
+author: >-
+  [Pablo Baeyens](https://github.com/mx-psi) (Datadog), [Tiffany
+  Hrabusa](https://github.com/tiffany76) (Grafana Labs), [Andrej
+  Kiripolský](https://github.com/andrejkiri) (Grafana Labs)
+sig: Contributor Experience
+cSpell:ignore: Andrej Baeyens Contribfest gatekeeping Hrabusa Kiripolský
+---
+
+The OpenTelemetry Contributor Experience SIG recently surveyed the community to
+learn what it's like to contribute to the project and what we can do to improve
+the contributor experience. The Contributor Experience Survey asked contributors
+for their thoughts about the project's organization, SIG contributions,
+leadership committees interactions, and event attendance. We received 120
+responses, which we'll use to make contributing to OpenTelemetry easier and more
+rewarding. A big thank you to everyone who participated in the survey! Let's
+review the results.
+
+## Key takeaways
+
+We conducted this survey to learn what the project can do better. Here are the
+key things we learned, along with areas we can improve for contributors:
+
+- **Getting started**: 48% of newer contributors don't confidently know how to
+  get started. Intra- and inter-SIG communication and contributing guides need
+  improvement.
+- **SIG meetings**: 82% of contributors find SIG meetings useful. Meeting
+  attendance correlates to greater knowledge about navigating and contributing
+  to the project, but some dislike the synchronicity meetings require.
+- **Workflows**: More than half of contributors do not receive timely feedback
+  on their contributions. They also experience project tooling headaches and
+  gatekeeping bottlenecks.
+- **Project and subproject priorities**: Less than half of maintainers can
+  confidently say they know the priorities of OpenTelemetry's leadership
+  committees. Contributors also say that it's difficult to find subproject
+  priorities.
+- **Events**: Contributors want more in-person events that are focused on
+  collaboration rather than talks.
+
+## Detailed insights
+
+With a total of 42 questions, this survey covered the breadth of the
+OpenTelemetry contributor experience. In this section, we analyze the responses
+and summarize what we learned.
+
+In some categories, the sample size is quite small and therefore the margin of
+error is high. We have included the number of responses in parentheses wherever
+possible.
+
+### About the respondents
+
+We heard from 120 contributors, including 45 maintainers and at least one
+contributor from each SIG. Here's more about our respondents:
+
+- 63% work for an observability vendor.
+- 20% have been contributing for less than six months.
+- 86% live in North America (45%) or EMEA (41%). This roughly matches the data
+  from devstats [TODO: add link], where we see 39% and 43% respectively.
```

Comment:
@mx-psi I think you might have added the devstats information here. Do you have a link we can point to? I couldn't find a devstats dashboard that shows those percentages. Thanks!

Reply:
There is no devstats link that directly shows this, I had to use a Python script and manually classify each country into its own region. We did present a graph with this information historically on the last Project Update talk at KubeCon EU 2025 (see graph [here](https://youtu.be/nclJn1KEjis?t=1487), 24:47 minute mark)